packages/components/src/styles/_content.scss

```diff
@@ -18,11 +18,13 @@
   [lang='ara'] {
     // `Important` below is required to override aggressive styling on pages
     font-family: var(--alpheios-arabic-font-face) !important;
+    font-variant-ligatures: normal;
   }
 
   [lang='syr'] {
     // `Important` below is required to override aggressive styling on pages
     font-family: var(--alpheios-syriac-font-face) !important;
+    font-variant-ligatures: normal;
```

Comment:
does this need an !important flag? And would it be a good idea to add it for everything not just the syriac font?

Reply:
```
font-variant-ligatures: normal;
```

In this case we don't need an important flag, because it doesn't have important on the site.
I believe that it seems something like use or not use other characters for headings.
Loeb has this styles
```
h1, h2, h3, h4, .hero-content h1 {
    font-family: "zephtextregular";
    font-variant-ligatures: none;
}
```

But I am not sure with all languages and this property - I haven't any experience with it
That's why I added only to `syr` and `ara` as they have specific fonts

I think if we want to add such a property to other languages - we first need to collect words for testing those language
@balmas , what do you think?

Reply:
Let's enter an issue for it and see if it comes up again before doing any more